EmailNotificationDaily/Functions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+
+namespace EmailNotificationDaily
+{
+    public class Functions
+    {
+        // This function will get triggered/executed when a new message is written 
+        // on an Azure Queue called queue.
+        public static void ProcessQueueMessage([QueueTrigger("queue")] string message, TextWriter log)

```

Comment:
@jSylvestre there are 3 identical files like this, what's the point of any of them?